scripts/travis_yml_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
We have *_test.py file for the test related to *.py file.

Reply:
Ok so this should be a lint test then?

Reply:
This should also be 2020 :)

Reply:
Done!